src/workerd/server/workerd.c++

```diff
@@ -1072,7 +1151,16 @@ private:
     kj::Own<const kj::ReadableFile> file;
   };
 
-  static kj::Maybe<ExeInfo> tryOpenExe(kj::StringPtr path) {
+#if _WIN32
+  static kj::Maybe<ExeInfo> tryOpenExe(kj::Filesystem& fs, kj::StringPtr path) {
+    auto parsedPath = fs.getCurrentPath().evalNative(path);
```

Comment:
I think the same comment as on Unix applies to Windows, so we should be using `CreateFile()` to open the path here. That said there would be a ton of boilerplate... I would accept a comment instead:

> TODO(bug): Like with Unix below, we should probably use native `CreateFile()` here, but it has sooooo many arguments, I don't want to deal with it.